pages.ru/common/echo.md

```diff
@@ -0,0 +1,24 @@
+# echo
+
+> Отобразить заданные аргументы.
+> Больше информации: <https://www.gnu.org/software/coreutils/echo>.
+
+- Отобразить текстовое сообщение. Примечание: кавычки необязательны:
+
+`echo "{{Привет, мир}}"`
+
+- Отобразить сообщение с переменной окружения:
+
+`echo "{{Мой путь - $PATH}}"`
+
+- Отобразить сообщение не перенося каретку на новую строку:
+
+`echo -n "{{Hello World}}"`
+
+- Добавить сообщение в файл:
+
+`echo "{{Hello World}}" >> {{file.txt}}`
```

Comment:
```suggestion
- Добавить сообщение в файл:

`echo "{{Привет, мир}}" >> {{путь/к/файлу.txt}}`
```

Almost the same issue as above.
Mandatory fix (error source: original page): [broken path](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): placeholder content conforming `path/to/*` glob expected.